### ISSUE TITLE
Manufacturer logo only saved in one file, is this the intended way since image rewrite?

### ISSUE THREAD

**Issue by therampagerado** [thirtybees/thirtybees#1859]
thirty bees version: edge as of today

Issue: Yesterday I added another manufacturer and I saw that my Warehouse theme does not display it's logo. Those were hardcoded to jpg, so easy fix for webp - all previous logos are showing but the new one does not.

In m folder I have:

![image](https://github.com/user-attachments/assets/3ee4322d-01d8-4709-ad7d-19e7b44732f2)

The logo is displayed in Manufacturers page in BO but there it's loaded from tmp folder:

![image](https://github.com/user-attachments/assets/fb615e38-0ccf-4594-8123-55f0f7645bd4)

Is this normal and if so how I can change this code to pull the correct image extension thirty bees offers:

<img class="imglog" alt="{$product_manufacturer->name}"  src="{$base_dir_ssl}img/m/{$product_manufacturer->id_manufacturer}-mf_image2.webp" />

Thanks!

**Comment by getdatakick** [comment]
If you are talking about source file, then yes -- it should exists only once, in the same format it was uploaded.

**Comment by therampagerado** [comment]
OK, but do we now generate the rest of the images in different formats.

Manufacturer 25 has only one webp image in /m/:

![image](https://github.com/user-attachments/assets/bc20dd71-fedd-4c09-9f78-2d2eeb0f51eb)

**Comment by getdatakick** [comment]
For me, the image types are correctly generated.
 
I've created new manufacturer, uploaded logo, and the image was saved correctly in all formats:

![image](https://github.com/user-attachments/assets/7223bd74-f8f2-4b23-91f6-947176d462ac)

Check image type definition, and make sure some of them are used by manufacturers

![image](https://github.com/user-attachments/assets/1ed20e03-8058-4049-860e-dbe6ad777979)

**Comment by therampagerado** [comment]
For my test store I have the following entities:

![image](https://github.com/user-attachments/assets/e790136e-bcfd-4ca1-be4f-06f456b17be0)


But for my production site where I have only Warehouse theme I don't have the majority of entities populated:

![image](https://github.com/user-attachments/assets/f3f1b998-ca14-4d13-aced-92aef785ccb7)

I think this is not correct and this is why it is not generating the proper files. How should I fix this? In both sites I just updated to the image rewrite PR and have not fixed anything manually.

**Comment by getdatakick** [comment]
There were few bugs related to migration of `tb_image_type` columns to `tb_image_entity_type` relationship table. I believe that now the process works correctly, but only from 1.5.1 to bleeding edge.

If you went through multiple bleeding-edge versions, it's likely that you were affected by those bugs. Please, manually re-save your image types, and associate correct image entities to them.

I've created new issue #1861 to make double-sure that migration works.

**Comment by therampagerado** [comment]
Thank you! Will sync the production shop to the entities in my test install and will check if this is my issue.

Yes, production is regularly updated in edge so probably I got caught there.
